Specialized Areas/Flow Actions/Remove HTML Tags from a String in a Flow/removeHtmlTags.js

```diff
@@ -1,4 +1,21 @@
-(function execute(inputs, outputs) {
-  var htmlString = inputs.htmlValue || '';
-  outputs.plainString = htmlString.replace(/<[^>]+>/g, '').trim();
-})(inputs, outputs);
+function removeHtmlTags(value) {
+  if (value === null || value === undefined) {
+    return '';
+  }
+
+  var stringValue = String(value);
+  var withoutTags = stringValue.replace(/<[^>]+>/g, ' ');
+
+  return withoutTags.replace(/\s+/g, ' ').trim();
+}
+
+if (typeof module !== 'undefined') {
+  module.exports = removeHtmlTags;
+}
+
+if (typeof inputs !== 'undefined' && typeof outputs !== 'undefined') {
+  (function execute(inputs, outputs) {
+    var htmlString = inputs.htmlValue || '';
+    outputs.plainString = removeHtmlTags(htmlString);
+  })(inputs, outputs);
+}
```

Comment:
You've completely changed the way this IIFE is configured and it's not a pattern that most ServiceNow devs will be familiar with. For a script action in a Flow it's best to stick with the system generated template.